examples/python/sample_mlc_chat.py

```diff
@@ -24,4 +24,4 @@
 )
 
 # Reset the chat module by
-# cm.reset_chat()
+# cm.reset_chat()
```

Comment:
Just a minor nit - maybe need a new line in the back.

Reply:
Hi, I think black automatically removes it, let me add it back